espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/drivers/AppDriver.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.espressoserver.lib.drivers
+
+import io.appium.espressoserver.lib.model.AppiumParams
+import io.appium.espressoserver.lib.model.BaseElement
+import io.appium.espressoserver.lib.model.Locator
+
+interface AppDriver {
+    val name: DriverContext.StrategyType
+    fun findElement(params: Locator): BaseElement
+    fun click(params: AppiumParams): Void?
```

Comment:
I assume we could replace Void? with Unit to avoid unnecessary returns. It is not necessary to do it in scope of this PR. Could be done separately.

Reply:
https://www.baeldung.com/kotlin/void-type

Reply:
makes sense. changed it since I tried it to learn this